README.md

```diff
@@ -1,16 +1,16 @@
-## electron-forge examples
+## Electron Forge Templates
 
-This repo helps you to get started with many of the popular web frameworks in Electron, by providing a ready-to-go template based on [electron-forge](https://github.com/electron-userland/electron-forge/).
+This repo helps you to get started with many of the popular web frameworks in Electron, by providing a ready-to-go templates for [electron-forge](https://github.com/electron-userland/electron-forge/).
 
 ## How do I try these out?
 
-All of the repos have the same build scripts. `cd` into the directory you're interested in, and type this to get started:
+All of these templates are **built-in** to `electron-forge`.  All you have to do is install forge, initialize your project and get coding!
```

Comment:
How is the built-in part going to work? Does that just mean that they're available on NPM?